pkg/server/handler.go

```diff
@@ -138,21 +138,11 @@ func (mc *MsgHandler) handleClientMessages(msg network.InMsg, token string) erro
 }
 
 func (mc *MsgHandler) handleRequestReplyCommands(msg network.InMsg, token string) error {
-	replyTo, ok := msg.Headers["reply_to"].(string)
-	if !ok {
-		return errors.New("reply_to property not provided")
-	}
-
-	corrID, ok := msg.Headers["correlation_id"].(string)
-	if !ok {
```

Comment:
Should we still validate if `replyTo` and/or `correlationId` were correctly passed?

Reply:
I think this should be validated just in the interactors so the client can be notified if there is some error on these parameters.
See my commit fix about it: 8c22e848d02d95c0f06e8655d24da1150bddcf7e

Reply:
I this function we already know that reply_to is mandatory information to RPC commands, maybe we can check if the reply_to is empty here.
Another thing to stay tuned. correlation_id is it mandatory information to RPC?

Reply:
I think we only should enforce the usage of `reply_to` as you said and `correlation_id` continues being optional.

Reply:
> I think this should be validated just in the interactors so the client can be notified if there is some error on these parameters.
> See my commit fix about it: [8c22e84](https://github.com/CESARBR/knot-babeltower/commit/8c22e848d02d95c0f06e8655d24da1150bddcf7e)

I'm trying to avoid moving this publishing to the interactors (it's already tied with AMQP details), so I think `controller` is the best layer (as you did in your patch).

Reply:
As @ramonhpr does this validation in his PR, I will close this discussion and approve this one.